libs/astarte_data_access/lib/astarte_data_access/fdo/ownership_voucher.ex

```diff
@@ -0,0 +1,38 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.DataAccess.FDO.OwnershipVoucher do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Astarte.DataAccess.FDO.OwnershipVoucher
+  use TypedEctoSchema
+
+  @primary_key {:id, Ecto.UUID, autogenerate: true}
+  typed_schema "ownership_vouchers" do
+    field :private_key, :binary
+    field :voucher_data, :binary
+    field :expiry, :integer
```

Comment:
explicit is better than implicit

```suggestion
    field :expiry, Astarte.DataAccess.DateTime
```